tests/isolated/patcher_threading_subclass_done.py

```diff
@@ -0,0 +1,40 @@
+import queue
+import threading
+
+
+class Worker(threading.Thread):
```

Comment:
This is defined _before_ monkeypatching, so it creates an OS thread, is that what you intended?

Reply:
The Thread class is never monkey-patched. It's the underlying `_thread` methods that are.